index.js

```diff
@@ -46,11 +47,40 @@ class ConventionalChangelog extends Plugin {
       if (increment && semver.valid(increment)) {
         return increment;
       } else if (isPreRelease) {
-        const type =
-          releaseType && (options.strictSemVer || !semver.prerelease(latestVersion))
-            ? `pre${releaseType}`
-            : 'prerelease';
-        return semver.inc(latestVersion, type, preReleaseId);
+        if (releaseType && (options.strictSemVer || !semver.prerelease(latestVersion))) {
+          return semver.inc(latestVersion, `pre${releaseType}`, preReleaseId);
+        }
+
+        const tags = await gitSemverTags({
+          lernaTags: !!options.lernaPackage,
+          package: options.lernaPackage,
+          tagPrefix: options.tagPrefix,
+          skipUnstable: true,
+          cwd: options.cwd
+        });
```

Comment:
Ideally the tag could be returned by `conventionalRecommendedBump` but it's not...